usaspending_api/disaster/v2/urls.py

```diff
@@ -11,6 +12,7 @@
 
 urlpatterns = [
     url(r"^overview/$", OverviewViewSet.as_view()),
+    url(r"^award/count/$", AwardCountViewSet.as_view()),
```

Comment:
Would be nice to add the endpoint in alphabetical order since other devs like to keep lists organized when possible